### DISCUSSION TITLE
More of a request or possibility

### DISCUSSION THREAD

**Discussion by formytech** [ZoeyVid/NPMplus#741] [Q&A] [answered]
Is there a way for wildcard dns, or adding bind and making npmplus handle dns? It's so time-consuming bouncing back and forth, and with that it may also be possible to use it for HA

**Comment by Zoey2936** [comment, marked as answer]
please mention this in #731, but wildcard is possible, but that has nothing to do with NPMplus you can do wirldcard with any webserver

**Comment by formytech** [comment]
right at the moment, its driving me nuts having to make a dns name for every ip6 address because they don't work directly in npm, it's like it thinks the pattern is a domain instead.  the original was like that also, that and the wildcard thing had always been my mad point, but i swore dns had to be be in the same place the web server was for wildcard to work.

________________________________

**Reply by Zoey2936** [reply]
Can you send a screenshot of a not working ipv6 config?
